src/main/java/org/radarbase/management/config/ManagementPortalSecurityConfigLoader.kt

```diff
@@ -42,21 +42,6 @@ class ManagementPortalSecurityConfigLoader {
 
     private var isAdminIdCreated: Boolean = false
 
-    /**
-     * Resets the admin password to the value of managementportal.common.adminPassword value if
-     * exists.
-     */
-    @EventListener(ContextRefreshedEvent::class)
```

Comment:
The overrideAdminPassword() method is no longer coupled to the ContextRefresjed event. Is this intended?